spec.html

```diff
@@ -34119,26 +34119,26 @@ <h1>Properties of Array Iterator Instances</h1>
             </tr>
             <tr>
               <td>
-                [[IteratedObject]]
+                [[IteratedArrayLike]]
               </td>
               <td>
-                The object whose array elements are being iterated.
+                The array-like object that is being iterated.
               </td>
             </tr>
             <tr>
               <td>
-                [[ArrayIteratorNextIndex]]
+                [[ArrayLikeNextIndex]]
               </td>
               <td>
-                The integer index of the next integer index to be examined by this iteration.
+                The integer index of the next element to be examined by this iterator.
               </td>
             </tr>
             <tr>
               <td>
-                [[ArrayIterationKind]]
+                [[ArrayLikeIterationKind]]
               </td>
               <td>
-                A String value that identifies what is returned for each element of the iteration. The possible values are: ~key~, ~value~, ~key+value~.
+                A String value that identifies what is to be returned for each element of the iteration. The possible values are: ~key~, ~value~, ~key+value~.
```

Comment:
imo the original is identically as correct as this change; the change seems fine tho.

Reply:
I've also removed "to be" from descriptions of `[[MapIterationKind]]` and `[[SetIterationKind]]` in 5226b3c.